src/components/Notifications/NotificationGroupCard/NotificationGroupCard.tsx

```diff
@@ -60,40 +59,34 @@ function NotificationGroupCard(props: Props) {
       <Accordion defaultExpanded={defaultExpanded}>
         <AccordionSummary expandIcon={<ExpandMoreIcon />}>
           <AccordionSummaryContainer>
-            {isLoading ? (
-              <Skeleton animation="wave" width={100} height={20} data-testid={NOTIFICATION_CARD_LOADING_TEST_ID} />
-            ) : (
-              <AccordionTitle data-testid={NOTIFICATION_CARD_TITLE_TEST_ID}>
-                {t(`settings.notifications.subscription_group_label_${subscriptionGroupKeys}`)}
-              </AccordionTitle>
-            )}
-            {isLoading ? (
-              <Skeleton animation="wave" width={200} height={24} />
-            ) : (
-              <AccordionDescription data-testid={NOTIFICATION_CARD_DESCRIPTION_TEST_ID}>
-                {t(`settings.notifications.subscription_group_description_${subscriptionGroupKeys}`)}
-              </AccordionDescription>
-            )}
+            <Title variant="h6" data-testid={NOTIFICATION_CARD_TITLE_TEST_ID}>
+              {t(`settings.notifications.subscription_group_label_${subscriptionGroupKeys}`)}
+            </Title>
+            <Description data-testid={NOTIFICATION_CARD_DESCRIPTION_TEST_ID}>
+              {t(`settings.notifications.subscription_group_description_${subscriptionGroupKeys}`)}
+            </Description>
           </AccordionSummaryContainer>
         </AccordionSummary>
-        <AccordionDetails>
-          {notificationTypesInGroup.map(type => (
-            <NotificationItemContainer key={type}>
-              <NotificationItemTextIconContainer>
-                <NotificationIcon name={type} />
-                <NotificationItemText>{t(`settings.notifications.types.${type}`)}</NotificationItemText>
-              </NotificationItemTextIconContainer>
-              {!disabled && (
-                <Switch
-                  onChange={(_, checked) => handleOnChangeNotificationSetting(checked, type)}
-                  checked={subscriptionDetails.messageType[toCamel(type)].email}
-                  disabled={isSavingSubscription}
-                  data-testid={NOTIFICATION_CARD_SWITCH_TEST_ID}
-                />
-              )}
-            </NotificationItemContainer>
-          ))}
-        </AccordionDetails>
+        {!isLoading && (
```

Comment:
Won't we be showing any kind of loader?

Reply:
Yes but not for the text. Only disabling input and loading button